reason-react-tutorial.md

```diff
@@ -0,0 +1,681 @@
+
+If you're still annoyed by having to write `{ReasonReact.string(description)`, you can create a `<Str string={description} />` component that just wraps `ReasonReact.string()`. This might feel a little less awkward and boils down to essentially the same thing.
+
+We're done! This is what our completed `<Card />` component looks like.
+
+{{< highlight javascript >}}
+let component = ReasonReact.statelessComponent("Card");
+
+let make = (~name, ~description, ~href, _children) => {
+  ...component,
+  render: _self =>
+    <div>
+      <h3>
+        <a href target="_blank" rel="noopener noreferrer">
+          {ReasonReact.string(name)}
+        </a>
+      </h3>
+      <p> {ReasonReact.string(description)} </p>
+    </div>,
+};
+{{< /highlight >}}
+
+Let's integrate it into our app so we can see it in action. If you cloned the repo head on over to `src/App.re`. You'll notice it's pretty bare right now.
+
+{{< highlight javascript >}}
+let component = ReasonReact.statelessComponent("App");
+
+let make = _children => {
+  ...component,
+  render: _self => <div> {ReasonReact.string("Hello world!")} </div>,
+};
+{{< /highlight >}}
+
+Let's replace the "Hello world!" text with our `<Card />` component. We'll have to add some fake props since we haven't added real data just yet (don't worry, we'll get there soon).
+
+{{< highlight javascript hl_lines="7" >}}
+let component = ReasonReact.statelessComponent("App");
+
+let make = _children => {
+  ...component,
+  render: _self =>
+    <div>
+      <Card name="reason" href="" description="This is cool" />
+    </div>,
+};
+{{< /highlight >}}
+
+We also didn't need to import our `<Card>` because every file in Reason is automatically a module in the global namespace. This takes a little getting used to, but I've found that the Reason/OCaml module system can be quite elegant. Not having to explicitly define an import path makes it easy to move files around or update folder structure later on. It also makes it easier to just pull in the modules you need without interrupting your workflow when you're in the middle of a project.
+
+When we check out the page we can see that our card is indeed on the page correctly, although it's a little bare.
+
+![image of bare card](https://res.cloudinary.com/da2iq7dge/image/upload/v1543390295/Screen_Shot_2018-11-25_at_11.17.07_PM_cpo0mi.png)
```

Comment:
If the picture doesn't add much value, I would leave it out to keep the page performance up. I think in this case people have followed you and see the same result or can imagine how it would look like. If the thing gets more complex, I think adding a screenshot it fine.